zhaquirks/sinope/thermostat.py

```diff
@@ -645,3 +646,93 @@ class SinopeG2Thermostats(SinopeTechnologiesThermostat):
             }
         }
     }
+
+
+class SinopeHPThermostats(SinopeTechnologiesThermostat):
```

Comment:
I don't see any real reason this needs to extend/inherit from `SinopeTechnologiesThermostat`, instead of just `CustomDevice`. We should probably change it on all Sinope thermostat quirks in a future PR.

For now, I guess it's fine, as the others already do it.